webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -151,7 +152,11 @@ const HostDetails = ({
         >
           {tabs &&
             tabs.map(tab => (
-              <Tab key={tab} eventKey={tab} title={tab}>
+              <Tab
+                key={tab}
+                eventKey={tab}
+                title={TranslateMapper.getTranslatedString(tab) || tab}
```

Comment:
According to Patternfly-react, this `should be a <TabTitleText> and/or <TabTitleIcon> for proper styling.`